Dockerfile

```diff
@@ -11,15 +11,26 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
+FROM openzipkin/jre-full:1.8.0_72
+MAINTAINER OpenZipkin "http://zipkin.io/"
 
-FROM openzipkin/zipkin-base:base-1.40.1
+RUN apk update && apk add curl
 
-MAINTAINER OpenZipkin "http://zipkin.io/"
+ENV ZIPKIN_REPO https://jcenter.bintray.com
+ENV ZIPKIN_VERSION 0.20.1
 
-ENV ZIPKIN_JAVA_VERSION 0.20.1
+# Use to set heap, trust store or other system properties.
 ENV JAVA_OPTS -Djava.security.egd=file:/dev/./urandom
 
-RUN curl -SL $ZIPKIN_REPO/io/zipkin/java/zipkin-server/$ZIPKIN_JAVA_VERSION/zipkin-server-$ZIPKIN_JAVA_VERSION-exec.jar > zipkin-server.jar && \ 
+RUN mkdir /zipkin
+# Add environment settings for supported storage types
+# TODO create one for "mem" and "elasticsearch"
+ADD .cassandra_profile /zipkin/.cassandra_profile
+ADD .mysql_profile /zipkin/.mysql_profile
+
+WORKDIR /zipkin
+
+RUN curl -SL $ZIPKIN_REPO/io/zipkin/java/zipkin-server/$ZIPKIN_VERSION/zipkin-server-$ZIPKIN_VERSION-exec.jar > zipkin-server.jar && \
```

Comment:
thought of something.. we could add installing curl to the base java image, since we need it on all derivatives. Then.. we don't have an extra layer here. wdyt? https://github.com/openzipkin/docker-jre-full/blob/master/Dockerfile.in